installer/bash-installer

```diff
@@ -217,6 +217,6 @@ if [ "${custom_dir}" == "false" ]; then
     output "\nOr add the following line to your shell configuration file:" "info"
     output "  export PATH=\"\$HOME/${CLI_CONFIG_DIR}/bin:\$PATH\""
     output "\nOr install it globally on your system:" "info"
-    output "  mv ${binary_dest}/${CLI_EXECUTABLE} /usr/local/bin/${CLI_EXECUTABLE}"
+    output "  sudo mv ${binary_dest}/${CLI_EXECUTABLE} /usr/local/bin/${CLI_EXECUTABLE}"
```

Comment:
Using `sudo` might not be necessary. As this is just an instruction, maybe we can add a line below saying that sometimes one might need to use sudo?

Reply:
Yes. I'm also fine with that :)